### PR TITLE
Use easyrsa from Github

### PR DESCRIPTION
**Pull request by upodroid**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Easy RSA is being pulled from gs://kubernetes-release bucket instead of https://github.com/OpenVPN/easy-rsa. They publish their tar files as GitHub releases that we can consume easily.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cluster/gce/util.sh [hunk 1]
```diff
@@ -1781,23 +1781,21 @@ function setup-easyrsa {
   # Note: This was heavily cribbed from make-ca-cert.sh
   (set -x
     cd "${KUBE_TEMP}"
-    curl -L -O --connect-timeout 20 --retry 6 --retry-delay 2 https://dl.k8s.io/easy-rsa/easy-rsa.tar.gz
-    tar xzf easy-rsa.tar.gz
-    mkdir easy-rsa-master/kubelet
-    cp -r easy-rsa-master/easyrsa3/* easy-rsa-master/kubelet
```
Comment (BenTheElder) [inline cluster/gce/util.sh:1787]: Are we sure we're not using this anywhere?

Comment (upodroid) [reply cluster/gce/util.sh:1787]: We aren't, kubelet certificates are created using cfssl

**Comment by dims** [comment]
/approve
/lgtm
/hold

@upodroid please remove hold if we still need this

**Pull request merged**